app/enterprise/1.3-x/systemd.md

```diff
@@ -47,7 +47,9 @@ $ sudo systemctl status kong-enterprise-edition
 
 ## Customize the Kong Enterprise unit file
 
-The official systemd service is located at `/lib/systemd/system/kong-enterprise-edition.service`. For scenarios where customizations are needed (for example, configuring Kong or modifying the service file behavior), we recommend to create another service at `/etc/systemd/system/kong-enterprise-edition.service` to avoid conflicts upon reinstalling or upgrading Kong Enterprise.
+Once the Kong Enterprise installation is finished, the official systemd service will be located at `/etc/kong/kong-enterprise-edition.service`. For convenience, this file will be copied to `/lib/systemd/system/kong-enterprise-edition.service` by default if the directory `/lib/systemd/system/` exists. 
+
+For scenarios where customizations are needed (for example, configuring Kong or modifying the service file behavior), we recommend to create another service at `/etc/systemd/system/kong-enterprise-edition.service` to avoid conflicts upon reinstalling or upgrading Kong Enterprise.
```

Comment:
```suggestion
For scenarios where customizations are needed (for example, configuring Kong or modifying the service file behavior), we recommend creating another service at `/etc/systemd/system/kong-enterprise-edition.service` to avoid conflicts upon reinstalling or upgrading Kong Enterprise.
```